kubeflow/groceries_real_train.py

```diff
@@ -24,8 +24,9 @@ def train_pipeline(
     config_file: str = (
         "datasetinsights/configs/faster_rcnn_groceries_real.yaml"
     ),
-    epochs: int = 50,
+    extra_options: str = "",
```

Comment:
This is not implemented. Can you remove them to reduce confusion?

Reply:
I can remove this although i think its a large benefit to support it at some point